src/care/host_device_ptr.h

```diff
@@ -398,6 +398,7 @@ namespace care {
       }
    }; // class host_device_ptr
 
+#if !defined(CARE_ENABLE_IMPLICIT_CONVERSIONS)
    /////////////////////////////////////////////////////////////////////////////////
```

Comment:
Two things: 1) I wouldn't surround this by CARE_ENABLE_IMPLICIT_CONVERSIONS, this function could be used to print a host_device_ptr in any case
2) In the description below, could you please change "this line is needed for..." to "this function is needed for..."

Reply:
Actually you can ignore (1), but add a TODO comment to remove the CARE_ENABLE_IMPLICIT_CONVERSIONS when chai is updated

Reply:
(1) I removed the macro guard but used .getPointer instead of .data since there's no tagged release of chai with the .data method in it yet.
(2) Fixed.